_config.yml

```diff
@@ -1,4 +1,4 @@
-title: 18F Front End Guide
+title: 18F Front-End Guide
```

Comment:
I doubt that anyone is tracking the analytics on this closely, but I do think this will create a break in the data. That shouldn't be a problem though, right?

Reply:
I have no idea or know how we'd find out.

Reply:
I pinged Lauren. She said since the path and url aren't changing the reporting _should_ be fine. So I don't think there's anything to worry about.